src/components/FilterDrawer/FilterDrawer.jsx

```diff
@@ -1,25 +1,38 @@
 import React from 'react';
 
 import Drawer from 'antd/lib/drawer';
+import Typography from 'antd/lib/typography';
 
 import DateRangeForm from '../DateRangeForm/DateRangeForm';
+import EditTrackerForm from '../EditTrackerForm/EditTrackerForm';
+import AddTrackerForm from '../AddTrackerForm/AddTrackerForm';
+
+const { Title } = Typography;
 
 class FilterDrawer extends React.Component {
 	render() {
 		return (
 			<div>
 				<Drawer
-					title="Filter Results by Time Range"
+					title="Edit Tracker Properties"
 					placement="right"
 					closable={false}
 					onClose={this.props.toggleDrawer}
 					visible={this.props.drawerVisible}
-					width={'350px'}
+					width={'300px'}
 				>
 					<DateRangeForm
 						handleDateFilter={this.props.handleDateFilter}
 						handleDrawer={this.props.toggleDrawer}
 					/>
+					<Title level={4}>Edit Trackers</Title>
+					<EditTrackerForm
+						deerStates={this.props.deerStates}
+						updateState={this.props.updateState}
+						finishedLoading={this.props.finishedLoading}
```

Comment:
why use a `finishedLoading` indicator instead of loading? Or is there another reason to have that?